distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Feature: [#15642] Track design placement can now use contruction modifier keys (ctrl/shift).
 - Change: [#22596] Land ownership fixes described by .parkpatch files are now only considered on scenarios.
+- Change: [#22740] Add virtual floor to shifted track design placement.
```

Comment:
Change or Feature? I'd say the latter fits best, but your call!

Reply:
My reasoning was the feature is the actual track design placement being shiftable in the first place, which is already new this month. The extension of the virtual floor to track designs is notable, but a minor changes in comparison, I think.